addons/SimpleTODO/TODOItem.gd

```diff
@@ -4,18 +4,184 @@ extends HBoxContainer
 onready var text_field = $Text
 onready var label_hack = $Text/HackLabel
 
-signal delete
+var item_placement_holder: Panel
+var main: Control
+var undo_redo: UndoRedo
+
+var initial_item_index = 0
+
+# The main TODO scene column container node
+var main_column_container: HBoxContainer
+
```

Comment:
Why the empty lines

Reply:
I'm not sure 🤷, will change this? I wonder if there is a formatter for gdscript.